crates/logic/src/player/mod.rs

```diff
@@ -89,8 +90,9 @@ impl FalconPlayer {
         let elapsed = self.time.elapsed().as_secs();
         self.connection.execute_sync(move |connection| {
             connection.handler_state_mut().set_last_keep_alive(elapsed);
-            // TODO: remove ignore
-            connection.send_packet(elapsed as i64, falcon_send::write_keep_alive).ignore();
+            connection.send_packet(elapsed as i64, falcon_send::write_keep_alive)?;
+
+            Ok(())
```

Comment:
Wouldn't this be more concise?
```suggestion
            connection.send_packet(elapsed as i64, falcon_send::write_keep_alive)
```

Reply:
The `?` does some error conversion stuff here. It is necessary.

Reply:
Same here?
```suggestion
            Ok(connection.send_packet(elapsed as i64, falcon_send::write_keep_alive)?)
```